clusterloader2/pkg/config/cluster.go

```diff
@@ -53,6 +53,7 @@ type ClusterConfig struct {
 	// using kubernetes client. If false, clusterloader will avoid collecting kube-apiserver profiles.
 	APIServerPprofByClientEnabled bool
 	KubeletPort                   int
+	EnableMeasurementLogging      bool
```

Comment:
Actually - this really goes against where we would like to go: https://github.com/kubernetes/perf-tests/issues/980

Reply:
How is the field “EnableLogging” in the measurement option structure supposed to be used? It’s hardcoded and always true. There should be a way to set it false. 

If v(2) is preferred, how about simply setting the default  to 2 in api/default.go?

Reply:
It's not supposed to be logged - I/we would like to get rid of it and rely on verbosity flags for klog (exactly what you started doing).

I don't think we should be setting the --v in the code, that seems pretty misleading to users.
But I just looked into it, and it seems all our usage is going via this script:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/run-e2e.sh#L37

So if you set it there explicitly, that should work fine for us.

Reply:
Thanks, I’ll take a look.